draft-ietf-trans-rfc6962-bis.md

```diff
@@ -552,14 +552,12 @@ Tree no later than a fixed amount of time, known as the Maximum Merge Delay
 (MMD), after the issuance of the SCT. Periodically, the log MUST append all its
 new entries to its Merkle Tree and sign the root of the tree.
 
-Log operators MUST NOT impose any conditions on retrieving or sharing data from
-the log.
-
 ## Accepting Submissions
 
 Before accepting a submitted certificate or precertificate, the log MUST verify
-that it has a valid signature chain to an accepted trust anchor, using the chain
-of intermediate CA certificates provided by the submitter. Logs SHOULD accept
+that it has a valid signature chain to an accepted trust anchor, using the
+chain of intermediate CA certificates provided by the submitter. (This helps
+protect logs from being overloaded by invalid submissions.) Logs SHOULD accept
```

Comment:
That bit is also explained in the introduction - don't mind replicating it here, but seems a bit strange that there's a whole sentence in braces? 
How about:
using the
"...of intermediate CA certificates provided by the submitter. Logs SHOULD accept		 +chain of intermediate CA certificates provided by the submitter, to help protect logs from being overloaded by invalid submissions".